### DISCUSSION TITLE
windows terminal icon

### DISCUSSION THREAD

**Discussion by sfaelbore** [vitest-dev/vitest#5498] [Q&A]
i was using windows but seems like `check` icon is not render properly, it renders as "?" is theres a way i can configure it? or do i need to download a font? thanks

**Comment by hi-ogawa** [comment]
Can you provide screenshot just in case? I'm not sure which check mark you're talking about.

We probably use very common unicode, so you might just need to update or install some fonts.
For example, what do you see if you run this?

```sh
$ node -e 'console.log("can you see this check mark? \u2713")'
can you see this check mark? ✓
```

**Reply by sfaelbore** [reply]
@hi-ogawa yes i cannot see that check mark, is there any config that i missed? or maybe a font i need to install? 
![image](https://github.com/vitest-dev/vitest/assets/161677422/63828908-7925-4547-989b-0394bb48c131)

**Reply by hi-ogawa** [reply]
I'm not familiar with Windows environment, so you probably need to consult other places.
For example, it looks like it's a known issue of command prompt https://github.com/nodejs/node-v0.x-archive/issues/7940

**Reply by sfaelbore** [reply]
Sure, Im using Angular with there cli seems like the checkmark works fine the issue is with vitest, anyway if i have time, i will look into it or create issue for it.

**Reply by hi-ogawa** [reply]
Yeah, ideally Vitest could use some fallback like https://github.com/sindresorhus/figures, but I don't think we'll try to eagerly implement that since this looks like a very minor use case. If you have a reference about how angular cli does it, then that would be still nice though.
